tests/launchsecurity/sev.go

```diff
@@ -15,26 +22,79 @@ import (
 var _ = Describe("[sig-compute]AMD Secure Encrypted Virtualization (SEV)", func() {
 	BeforeEach(func() {
 		checks.SkipTestIfNoFeatureGate(virtconfig.WorkloadEncryptionSEV)
-		checks.SkipTestIfNotSEVCapable()
 	})
 
-	It("should start a SEV VM", func() {
-		const secureBoot = false
-		vmi := libvmi.NewFedora(libvmi.WithUefi(secureBoot), libvmi.WithSEV())
-		vmi = tests.RunVMIAndExpectLaunch(vmi, 240)
-
-		By("Expecting the VirtualMachineInstance console")
-		Expect(console.LoginToFedora(vmi)).To(Succeed())
-
-		By("Verifying that SEV is enabled in the guest")
-		err := console.SafeExpectBatch(vmi, []expect.Batcher{
-			&expect.BSnd{S: "\n"},
-			&expect.BExp{R: console.PromptExpression},
-			&expect.BSnd{S: "dmesg | grep --color=never SEV\n"},
-			&expect.BExp{R: "AMD Memory Encryption Features active: SEV"},
-			&expect.BSnd{S: "\n"},
-			&expect.BExp{R: console.PromptExpression},
-		}, 30)
-		Expect(err).ToNot(HaveOccurred())
+	Context("device management", func() {
+		var (
+			virtClient kubecli.KubevirtClient
+			nodeName   string
+			err        error
+		)
+
+		BeforeEach(func() {
+			virtClient, err = kubecli.GetKubevirtClient()
+			Expect(err).ToNot(HaveOccurred())
+
+			nodeName = tests.NodeNameWithHandler()
+			Expect(nodeName).ToNot(BeEmpty())
+
+			// Create a fake SEV device
+			mknodCmd := []string{"mknod", "/dev/sev", "c", "10", "124"}
```

Comment:
Can we first check, by running a command on virt-handler, if the node has the sev device already? If so, we would not create and not remove it after the test.

Reply:
Something simple from virt-handler would do, like `ls /proc/1/root/dev/sev`.

Reply:
Agree, the check makes sense. Addressed.